src/common/utils/replace_reference_with_alias.py

```diff
@@ -0,0 +1,36 @@
+from domain_classes.dependency import Dependency
+
+
+def is_alias(path: str, dependencies: list[Dependency]):
+    NUM_SPLITS = 1
+    path_start = path.split(":", NUM_SPLITS)[0]
```

Comment:
This does not look right. Paths should never be aliases. Only absolute paths.
We need to check if we have an alias for an absolute path.

Reply:
ah, yes. 
The function parameter is badly named. 

maybe something like this is better?
![image](https://user-images.githubusercontent.com/69512295/209093828-4d22c4ca-c1ac-44d9-b247-4357d730ccb4.png)

Reply:
I dont understand why you need to check if it's an alias at all. We know it never will be any aliases in DMSS.
What makes sense to me is that we need a check to see if a "path" `has_dependency_alias()`. And if it does, replace the path with the alias.

Reply:
ah, you are right.

It is no longer needed after refactoring the function. 
(before it was needed because of the structure of the loops - looping through dependencies before looping through strings in the extends lists)